store/storetest/team_store.go

```diff
@@ -1566,19 +1609,21 @@ func testTeamStoreAnalyticsGetTeamCountForScheme(t *testing.T, ss store.Store) {
 		SchemeId:    &s1.Id,
 		DeleteAt:    model.GetMillis(),
 	}
-	_ = (<-ss.Team().Save(t4)).Data.(*model.Team)
+	_, err = ss.Team().Save(t4)
+	require.Nil(t, err)
 
 	count5 := (<-ss.Team().AnalyticsGetTeamCountForScheme(s1.Id)).Data.(int64)
 	assert.Equal(t, int64(2), count5)
 }
 
 func testTeamStoreGetAllForExportAfter(t *testing.T, ss store.Store) {
-	t1 := model.Team{}
+	t1 := &model.Team{}
```

Comment:
nit: for consistency with `testTeamStoreGetTeamMembersForExport`, I'd prefer not to get the reference to the team here (so `t1 := model.Team{}`) so both tests act alike